README.md

```diff
@@ -1,2 +1,14 @@
 # siite
-The Quodl website - where the latest information on Quodl, the science behind it and sign up links can be found.
+The Quodl website - where the latest information on Quodl, the science behind it
+and sign up links can be found.
+
+## Set up
+- Clone this repo onto your machine
```

Comment:
It might be good to add the clone command for this, something like in the [dwyl/hapi-socketio-redis-chat-example](https://github.com/dwyl/hapi-socketio-redis-chat-example#1-clone-the-repository)